library/Registry.md

```diff
@@ -0,0 +1,22 @@
+# Q# Package Registry
```

Comment:
```suggestion
# Q# Project Registry
```

Reply:
Now I'm less certain about this suggestion... settling on a specific terminology is something I think we need some consensus on.